scpd/lms/static/sass/_overrides.scss

```diff
@@ -0,0 +1,410 @@
+/****************************************************************************************************
+*   Stanford Overrides File
+*
+*   Contains all the CSS overrides for the Stanford Open Source MOOC Project
+*
+****************************************************************************************************/
+
+//-----------------
+//  Mixin Styles
+//-----------------
+@mixin login_register_h1_style {
```

Comment:
I think this is a mixin we added (not sure if upstream?) for old-style registration,
so I doubt it's still in use.

Can you confirm and delete?

On that note, we should audit, well, pretty much all of these selectors and see what's actually still in use,
removing the cruft.